README.md

```diff
@@ -280,9 +264,7 @@ for (int i = 0; i < 10; i++) {
 
 ## Control Structures
 
-Don't use multiple return statements.
-
-Always surround `if` bodies with curly braces if there is an `else`. Single-line `if` bodies without an `else` should be on the same line as the `if`.
+Always surround `if` bodies with curly braces. Single-line `if` bodies may be on the same line as the `if`, but must still include curly braces.
```

Comment:
Unbraced if was a mistake in C. I want to make it more clear that braces are mandatory.